README.md

```diff
@@ -1,2 +1,82 @@
 # velero-plugin
-HPE plugin for Velero
\ No newline at end of file
+
+HPE plugin for Velero.
+To take snapshots of HPE volumes through velero you need to install and configure the HPE Snapshotter plugin.
```

Comment:
To take snapshots of CSP volumes through Velero you need to install configure the CSP snapshotter plugin.